cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -1048,8 +1051,14 @@ protected void jvmPerformanceOptions(List<EnvVar> envVars) {
             sbj.setOrganizationName("io.strimzi");
             sbj.setCommonName(getName());
 
+            Map<String, String> sbjAltNames = new HashMap<>();
+            sbjAltNames.put("DNS.1", getServiceName());
+            sbjAltNames.put("DNS.2", String.format("%s.%s.%s.svc.%s", podName.apply(cluster, i), getServiceName(), namespace, STRIMZI_DNS_DOMAIN));
```

Comment:
The regular service doesn't route the pods. So this line should be IMHO only:
```
sbjAltNames.put("DNS.2", String.format(%s.%s.svc.%s", getServiceName(), namespace, STRIMZI_DNS_DOMAIN));
```

Reply:
RIght. I'll change that.